models/articles.model.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection");
+
+exports.readArticleById = (article_id) => {
+  let queryString = "SELECT * FROM articles";
+  const queryVals = [];
+  if (article_id) {
+    queryString += ` WHERE article_id = $1;`;
+    queryVals.push(article_id);
+  }
```

Comment:
I like the idea of making this function dynamic, but this is not behaviour that has been tested for and we should only be added logic that has been tested for, which in this case means we should always assume we are going to receive a article_id.